Yoast/wordpress-seo
# Add components css stylesheet to the plugin

## Context


* The JavaScript monorepo will now export a CSS file for the components package. Therefore, the plugin needs to be able to interpret that CSS.

## Summary


This PR can be summarized in the following changelog entry:

* [wordpress-seo] The plugin will now load the component's package's CSS.

## Relevant technical choices:

* We have chosen to extract the CSS to a single file using MiniCssExtractPlugin.

## Test instructions

This PR can be tested by following these steps:

* Build the plugin.
* Verify that on all pages where we need our styling (wp admin, post edit etc.) there is a script called `<link rel="stylesheet" id="yoast-seo-monorepo-css" href="http://basic.wordpress.test/wp-content/plugins/wordpress-seo/css/dist/monorepo-133-RC2.css?ver=13.3-RC2" media="all">` or something like it (simply search with cmd+f for `monorepo` in your console (should be in the header).
* If you want to see the CSS in action:
	* Add `yoast` to the classes on the `<body>` on a page where you expect our styling to be.
	* Further down, find some easy HTML section to edit in the console, and add `<button class="yoast-button yoast-button--primary">primary</button><button class="yoast-button yoast-button--secondary">secondary</button>` 

## UI changes

## Documentation

## Quality assurance


Fixes https://github.com/Yoast/javascript/issues/517
